### PR TITLE
Add Sprint Details

### PR DESCRIPTION
**Pull request by chessbyte**
Sprint Details are implemented as a Jekyll Collection backed by per-sprint data in a yaml file.

Addresses part of https://github.com/ManageIQ/manageiq.org/issues/833

### CODE REVIEW

File: site/_layouts/sprint.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+---
+layout: page
```
Comment (Fryguy) [inline site/_layouts/sprint.md:2]: Why is this page sprint.md if it's all html?  Is that a jekyll thing?

File: site/sprints/index.md [hunk 2]
```diff
@@ -3,8 +3,6 @@ layout: page
 title: Sprints
```
Comment (Fryguy) [inline site/sprints/index.md:3]: Same comment for the .md file buth with html in it.

File: site/_layouts/sprint.md (hunk 1 above)
Comment (chessbyte) [reply site/_layouts/sprint.md]: Addressed

File: site/sprints/index.md (hunk 2 above)
Comment (chessbyte) [reply site/sprints/index.md:3]: Resolved

**Pull request merged**